importer/importer/users.py

```diff
@@ -244,18 +244,19 @@ def confirm_practitioner(user, user_id):
 def create_roles(role_list, roles_max):
```

Comment:
can remove `roles_max` from the params here since it's not used any more?
from where it's called in main.py as well

Reply:
any update on this?